docs/tutorial-for-unity.md

```diff
@@ -0,0 +1,64 @@
+# Terminology for Unity Developers
+
+## What is an FBO?
+
+FBO stands for frame object buffer. It's equal to the renderer target that we generated.
+
+## What is a node?
+
+You can regard a node as a "Render Pass" in Unity.
+A node is customized code you can do whaterver you want to the render result, you can change the output of **InitialPostProcessingNode** like Tinting,
+you can also change the input of "HighPassNode" to make everying "blooming".
+you can also create a node rendering character's outline just outline, add to then combine it with  **InitialPostProcessingNode**'s output to create an XRay node.
+> this requires technology to rendereing specificed layer, I'm not sure whether Terasology has this feature, hopefully it would add the "layer" conecpt in the future.
+
+## How nodes work
+
+a node has Inputs and Outputs, it's input are result from previous node, and it's output can be next node's input.
+
+For example, highPass node use `_SceneColor` as input, then bloom node use both `_SceneColor` and highpass node's output as input.
+
+`_SceneColor` is the scene texture, I'm writing this style for better understanding for unity developers.
+highpass node is a node to grab light area then send to blur node to make the bright area of bloom.
+it's being used for **InitialPostProcessingNode** as input for final FBO which is the final output of the camera.
+
+## How to create a FBO
+
+in Unity we can use the code below 
+
+```C#
+RenderTexture  computeTex = new RenderTexture (IMG_WIDTH, IMG_HEIGHT, 24);
+
+```
+
+in Terasology we need to setup these at `setDependencies(Context context)`
+
+```java
+//the root Fbo for everything
+DisplayResolutionDependentFbo displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFbo.class);
+//create the fbo with scale
+FBO someFbo = requiresFbo(new FboConfig(FBO_URI, HALF_SCALE, FBO.Type.DEFAULT), displayResolutionDependentFBOs);
+//create the fbo with width and height
+FBO someFbo2 = requiresFbo(new FboConfig(FBO_URI, IMG_WIDTH, IMG_HEIGHT, FBO.Type.NO_COLOR), displayResolutionDependentFBOs);
+```
```

Comment:
Setting up FBOs in Terasology translates to Unity as setting up rendering targets:

```C#
RenderTexture  computeTex = new RenderTexture (IMG_WIDTH, IMG_HEIGHT, 24);
```

In Terasology, FBOs can be set up using `setDependencies(Context context)`:

```java
//the root Fbo for everything
DisplayResolutionDependentFbo displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFbo.class);
//create the fbo with scale
FBO someFbo = requiresFbo(new FboConfig(FBO_URI, HALF_SCALE, FBO.Type.DEFAULT), displayResolutionDependentFBOs);
//create the fbo with width and height
FBO someFbo2 = requiresFbo(new FboConfig(FBO_URI, IMG_WIDTH, IMG_HEIGHT, FBO.Type.NO_COLOR), displayResolutionDependentFBOs);
```